src/main/java/thePackmaster/actions/anomalypack/RitualSightAction.java

```diff
@@ -0,0 +1,66 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
+
+package thePackmaster.actions.anomalypack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.GameActionManager;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+
+import java.util.Iterator;
+
+public class RitualSightAction extends AbstractGameAction {
+    private static final UIStrings uiStrings;
+    public static final String[] TEXT;
+    private AbstractPlayer p;
+
+    public RitualSightAction() {
+        this.actionType = ActionType.CARD_MANIPULATION;
+        this.p = AbstractDungeon.player;
+        this.duration = Settings.ACTION_DUR_FAST;
+    }
+
+    public void update() {
+        if (this.duration == Settings.ACTION_DUR_FAST) {
+            if (this.p.hand.size() == 1) {
+                this.addToTop(new MakeTempCardInDrawPileAction(this.p.hand.getBottomCard().makeStatEquivalentCopy(),1,true,false));
```

Comment:
```suggestion
                this.addToTop(new MakeTempCardInDrawPileAction(this.p.hand.getBottomCard(),1,true,false));
```
MakeTempCard actions already make copies